guide/onboarding/funding-a-wallet.md

```diff
@@ -22,93 +30,205 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
-   image = "/assets/images/guide/onboarding/funding-a-wallet/buy-bitcoin.png"
-   retina = "/assets/images/guide/onboarding/funding-a-wallet/buy-bitcoin@2x.png"
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/lightning-services-invoice.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/lightning-services-invoice@2x.png"
+   layout = "shadow"
+   caption = "The first time the user funds their wallet a payment channel will be opened."
+   alt-text = "Screen showing funding the wallet with an invoice."
    width = 250
-   height = 550
-   alt-text = "Showing recovery phrase one word at a time"
-   caption = "Buy bitcoin screen from the [Wallet UI Kit](https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=1228%3A28550)"
-   layout = "float-left-desktop -background -shadow"
+   height = 541
 %}
 
-As noted, most exchanges offer the option to buy bitcoin using fiat currencies. Once purchased, users can transfer those funds to a separate, external wallet. This might confuse new users who don't understand the difference between exchange wallets and personal wallets.
-
-Bitcoin ATMs are another great option for new users to exchange cash for bitcoin in person. These machines also let users trade their bitcoin for cash, which can be useful if they are somewhere new or foreign.
+{% include picture.html
+   image = "/assets/images/guide/onboarding/funding-a-wallet/lightning-services-address.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/lightning-services-address@2x.png"
+   class = "shadow"
+   caption = "The user should have the option to fund via an address, as they may not have funds in Lightning."
+   alt-text = "Screen showing funding the wallet with an address."
+   width = 250
+   height = 541
+%}
 
-Finally, new users can buy bitcoin directly from other bitcoin users such as friends, other users at a meetup, or intermediary services like [localbitcoins](https://localbitcoins.com/).
+{% include picture.html
+     image = "/assets/images/guide/onboarding/funding-a-wallet/lightning-services-final.png"
+   retina = "/assets/images/guide/onboarding/funding-a-wallet/lightning-services-final@2x.png"
+   class = "shadow"
+   caption = "Once the wallet has been funded it is ready to use."
+   alt-text = "Screen showing the wallet funded and ready to use."
+   width = 250
+   height = 541
+%}
 
 </div>
 
-{% include /tip/open.html label="Tip: Teach users how to transfer" icon="info" color="blue" %}
+The initial screen when the user opens their wallet for the first time should prompt the user to fund their wallet so they can begin using it. Depending on how your application connects to the Bitcoin and Lightning network you may need to notify the user that funds will not appear until these connections are established and syncd. We recommend connecting to the Bitcoin network via Neutrino which offers a simple, private way to source blockchain data.
```

Comment:
Another thought. Should this be "a client like Neutrino" or "block filters"? I raise this point because I like to adhere to a principle of avoiding engineering recommendations in the Guide. From a design perspective, we recommend principles like privacy, transparency, interoperability, etc. I feel that how the bitcoin and lightning engineers accomplish these principles is for them to decide. Thoughts, anyone?

Reply:
I agree, how about saying something like below: 

Common ways applications connect to the Bitcoin network is using compact block filters (BIP 157/158) or SPV to query and download block headers. Block filters offer more privacy but take some initial time to sync whilst an SPV does not offer any privacy to users but is ready instantly. 

Can link to the entries for these in the `Nodes` page.